1 HW_Standard/HW_Probopass.yaml

```diff
@@ -0,0 +1,22 @@
+species: Probopass
+ingamename: SiIva
+displayname: SiIvaGunner
+setname: Halloween
+tags: [halloween]
+item: Thick Club
+ability: Snow Warning
+moves:
+    - [Spacial Rend]
+    - [Stone Edge]
+    - [Grass Whistle]
+    - [Rest]
```

Comment:
swap stone edge and spacial rend to keep the strongest move on top (as stone edge is STAB)